experiments_main.cpp

```diff
@@ -99,17 +98,37 @@ DEFINE_validator(hash_alg, &ValidateHashAlg);
 
 DEFINE_uint32(num_bins, 256, "Number of bins used to discretize, if --transform=disc");
 
-DEFINE_uint32(num_threads, 1, "number of OpenMP threads, default: 1, "
-                              "use --num_threads=0 to use all available cores");
+DEFINE_uint32(num_threads,
+              1,
+              "number of OpenMP threads, default: 1, "
+              "use --num_threads=0 to use all available cores");
 
 
-
-namespace fs = std::filesystem;
 using namespace ts;
 
 template <class char_type, class kmer_type, class embed_type>
 struct SeqGenModule {
```

Comment:
This is a misnomer that made sense originally but now it doesn't anymore. Maybe rename to ExperimentsHelper or something similar. I'd also make it a class for consistency with the rest of the code.

Reply:
Yeah, I'm revamping this entire class/file in the next PR so will do it there.